js/layout-javascript/news-feed-code.js

```diff
@@ -284,6 +285,41 @@ DynamicList.prototype.attachObservers = function() {
         }, 0);
       }
     })
+    .on('click change keydown', '.hidden-filter-controls-filter-date-range', function(event) {
```

Comment:
Most of the other event listeners only listen for the `click` and `keydown` events. Why is this listening to the `change` event too?

Reply:
we need _change_ event for the next situation:
**If the FROM date is updated to be later than to TO date (or the TO date is updated to be earlier than the FROM date), the TO (FROM) date is automatically updated to match the value of the FROM (TO) date**

Reply:
Where can I find the code that changes the TO date to match the FROM date (or vice versa)?